testdata/backup-management/mongodb/keyfile

```diff
@@ -0,0 +1,16 @@
+TPNL+k+XwsuTNRyYCldDkUXmO43Hl7k5sEFMDVUUS0omW8bsut6HCPyjL2bMEvoN
```

Comment:
this came from pbm e2e tests - we only use it as keyfile for mongo authentication, so I guess it's safe and exposes nothing 🙏